syntax/scan.go

```diff
@@ -98,6 +98,26 @@ const (
 	RETURN
 	WHILE
 
+	// Illegal tokens (reserved Python keywords that aren't part of the Starlark language)
```

Comment:
Let's avoid confusion with the name ILLEGAL above and document this simply as:

// Reserved words (following Python); unused in Starlark

then the tokens below don't need individual comments.